python/cuml/cuml/ensemble/randomforest_common.pyx

```diff
@@ -313,6 +313,15 @@ class BaseRandomForestModel(UniversalBase):
         self.dtype = np.float64
         self.update_labels = False
         super().cpu_to_gpu()
+        # Set fitted attributes not transferred by treelite, but only when the
+        # accelerator is active.
+        # We only transfer "simple" attributes, not np.ndarrays or DecisionTree
+        # instances, as these could be used by the GPU model to make predictions.
+        # Which means they need to be of the type that the GPU model expects.
+        if GlobalSettings().accelerator_active:
```

Comment:
Why not do this all the time? Why only when the accelerator is active?

Reply:
Because I'm not brave enough. I was hoping to tempt someone who understands the GPU<>CPU conversion for RFs to comment.

My thinking was that if doing this leads to breakage, at least it is contained to `cuml.accel` users :-/

Reply:
I think that's fair. Maybe check whether this particular code path is covered by existing tests. If that gives us enough confidence, then we can drop the guard.